Doc/library/io.rst

```diff
@@ -924,8 +925,8 @@ Text I/O
       locale encoding using :func:`locale.setlocale`, use the current locale
       encoding instead of the user preferred encoding.
 
-   :class:`TextIOWrapper` provides these members in addition to those of
-   :class:`TextIOBase` and its parents:
+   :class:`TextIOWrapper` provides these data attributes and method in
```

Comment:
```suggestion
   :class:`TextIOWrapper` provides these attributes and methods in
```

Reply:
I think we should keep the word "data" before "attributes". Attributes can be either data attributes or function attributes with corresponding methods: https://docs.python.org/3.7/tutorial/classes.html#instance-objects